credsweeper/filters/variable_not_allowed_pattern_check.py

```diff
@@ -1,36 +0,0 @@
-import re
```

Comment:
Why this filter has been removed?
There is no description for it..

Reply:
AWS_KEY_ID is filtered often :(
The feature was migrated to ML

Reply:
VariableNotAllowedPatternCheck mention was added to PR description

Reply:
Okay I see..
Is there a lot of patterns that do not allowed?
I think if we delegate this filter's role to ML, we will be harder to reasoning and debugging why.
So I'm worry about this.

Reply:
It seems filtering the credentials from ML will be better..
Okay let's move the role to ML.